hphp/runtime/vm/jit/vasm-ppc64.cpp

```diff
@@ -1644,6 +1680,138 @@ void fixVptrsForPPC64(Vunit& unit) {
 ///////////////////////////////////////////////////////////////////////////////
 } // anonymous namespace
 
+#include "hphp/util/logger.h"
```

Comment:
Oh sorry, that was part of debugging code so I could call Logger:: functions.

As you have discovered it works fine without it.


> On 6 Aug 2016, at 12:00 am, Gustavo Serra Scalet <notifications@github.com> wrote:
> 
> In hphp/runtime/vm/jit/vasm-ppc64.cpp <https://github.com/PPC64/hhvm/pull/143#discussion_r73696422>:
> 
> > @@ -1644,6 +1680,138 @@ void fixVptrsForPPC64(Vunit& unit) {
> >  ///////////////////////////////////////////////////////////////////////////////
> >  } // anonymous namespace
> >  
> > +#include "hphp/util/logger.h"
> @daxtens <https://github.com/daxtens> Why did you include this? After merging to new upstream code this week we had to remove this otherwise it doesn't compile
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/PPC64/hhvm/pull/143/files/cae8c83423507275bae9a5b5c4b77e5f2ad7061f#r73696422>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ADJ5BbVLtNTDAXN1xyyj5iaCcJUptAZRks5qc0IKgaJpZM4JFywq>.
>